### PR TITLE
[TRAVIS] Refactor GRAALVM tests

### PR DESCRIPTION
**Pull request by blackleg**
Refactor GRAALVM tests

### CODE REVIEW

**Comment by JaroslavTulach** [comment]
Work in #1746 is also touching the same `scripting.sh` file.

**Comment by blackleg** [comment]
@JaroslavTulach This pull request only changes the Travis config and disables tests for `platform/core.network` module that fails when runs in Travis VM.

File: nbbuild/travis/scripting.sh [hunk 1]
```diff
@@ -28,24 +28,15 @@ if [ -z "$GRAALVM" ]; then
   GRAALVM=`pwd`/$BASE
 fi
 
-# test on regular VM
-
-ant -f platform/api.scripting/build.xml test
```
Comment (JaroslavTulach) [inline nbbuild/travis/scripting.sh]: Should this really be removed?

Comment (blackleg) [reply nbbuild/travis/scripting.sh]: Tests on regular VM are executed in other Travis jobs like this: https://github.com/apache/netbeans/blob/master/.travis.yml#L362

**Comment by blackleg** [comment]
> Are the `libs.graalsdk`, `libs.graaljs`, `profiler.oql` & co. tests executed anywhere else?

Yes, they are executed in other Travis jobs.
- [platform/api.htmlui](https://github.com/apache/netbeans/blob/master/.travis.yml#L105)
- [platform/api.scripting](https://github.com/apache/netbeans/blob/master/.travis.yml#L110)
- [ide/libs.graalsdk](https://github.com/apache/netbeans/blob/master/.travis.yml#L251)
- [webcommon/libs.graaljs](https://github.com/apache/netbeans/blob/master/.travis.yml#L362)
- [profiler.oql](https://github.com/apache/netbeans/blob/master/.travis.yml#L348)

The tests for [platform/core.network](https://github.com/apache/netbeans/blob/master/.travis.yml#L123) doesn't work when they are executed runs in a Travis job and are disabled in config.

File: nbbuild/travis/scripting.sh [hunk 2]
```diff
@@ -28,24 +28,15 @@ if [ -z "$GRAALVM" ]; then
...
 $GRAALVM/bin/gu install R
 
-# test on GraalVM
+# Test on GraalVM
 
 JAVA_HOME=$GRAALVM ant -f platform/api.scripting/build.xml test
 JAVA_HOME=$GRAALVM ant -f ide/libs.graalsdk/build.xml test
 
-JAVA_HOME=$GRAALVM ant -f platform/core.network/build.xml test
+#JAVA_HOME=$GRAALVM ant -f platform/core.network/build.xml test
```
Comment (JaroslavTulach) [inline nbbuild/travis/scripting.sh]: I cannot integrate any change that doesn't work for `core.network`. There is a code that deals with proxy setup and it has to work on all supported JVMs.

Comment (blackleg) [reply nbbuild/travis/scripting.sh]: There is a problem with core.network module tests and Travis environment: https://github.com/apache/netbeans/pull/1773 and https://github.com/apache/netbeans/pull/1763.
 You could accept and merge this pull request to allow Travis run tests with GRAALVM for the rest of modules.

Comment (blackleg) [reply nbbuild/travis/scripting.sh]: Problems with `core.network` module tests and Travis env were fixed in this pull request: https://github.com/apache/netbeans/pull/1835

**Comment by matthiasblaesing** [comment]
Thank you.

**Comment by matthiasblaesing** [comment]
@backleg please don't use the github features, if your user account is not configured properly (properly means with a public visible email address and your real name). In this case the master merge has broken author information. Considering it is only a merge I don't consider it critical, but the github GUI is a mess regarding sane author information.

**Comment by jtulach** [comment]
Thanks a lot @blackleg for fixing the NetBeans+GraalVM testing!

**Pull request merged**